AudibleBoulders/AudibleBoulders
# Change gitSpyInit node script to bash script

#### What's this PR do?
Refactor gitSpyInit node script to bash script
Add "gitSpy/*" to .gitignore when you run "sh gitSpyInit"

#### What are the important parts of the code?
The new zipfile should match the files in DEV-gitSpy/

#### How should this be tested by the reviewer?
delete the post-commit, post-rewrite, and spyupdate file
also delete "gitSpy/*" from your .gitignore
replace your gitSpy folder with the new one
run "sh gitSpy/gitSpyInit" (no extension)
Check to see if all the correct files got generated
Run one more time and make sure the text isn't duplicated in the files

Insert your signature_hash and check to see if scripts still function as usual!
Thanks for testing!

#### Is any other information necessary to understand this?
N/A

#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
N/A

#### Screenshots (if appropriate)
N/A
